debian/copyright

```diff
@@ -154,7 +154,7 @@ License: BSD-3-clause-inih
 
 Files: src/plugins/ni/nickel-1.1.0/*
 Copyright: 2008 Charles Lindsay <charles@chaoslizard.org>
-License: BSD-3-clause-nickel
+License: Zlib
```

Comment:
Ohh, thanks for fixing this! Is it a completely unmodified Zlib license? Otherwise Zlib-nickel might be better.

Reply:
You're welcome. Stumbled upon it and at first thought it was BSD-3-clause as well :)
It's unmodified except for the copyright header...i'll change it to Zlib-nickel and include the content of the license file here.

Reply:
Wait...the license below does not have the header and copyright is included in the line above. I guess it's to leave it as "Zlib" fine then?

Reply:
> I guess it's fine to leave it as "Zlib"?

Sorry, I do not know, this file originally is from @pinotree

Reply:
Ok I did some reading on the topci and as long as the license text is unmodified an identifier (specified with `License: `) can be reused.